backend/tests/helpers/constants.py

```diff
@@ -3,7 +3,9 @@
 MNEMONIC = "test test test test test test test test test test test junk"
 MOCKED_PENDING_EPOCH_NO = 1
 MOCKED_FINALIZED_EPOCH_NO = 1
+MOCKED_EPOCH_NO_AFTER_OVERHAUL = 3
 MOCKED_CURRENT_EPOCH_NO = 2
+NO_PATRONS_REWARDS = 0
```

Comment:
I'm not sure about this - what's the point in adding this const if it's set to zero?

Reply:
NO_PATRONS_REWARDS is a const representing that there's lack of patrons_rewards. I've extracted it to use it in different formulas explicitly to roughly show that a given formula uses PATRONS_REWARDS value but in the cases it's exceptionally set to 0. 

Example:
MATCHED_REWARDS = TOTAL_REWARDS - ALL_INDIVIDUAL_REWARDS --- without NO_PATRONS_REWARDS
MATCHED_REWARDS = TOTAL_REWARDS - ALL_INDIVIDUAL_REWARDS + NO_PATRONS_REWARDS ---- now, with NO_PATRONS_REWARDS

WDYT?